File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/BalanceCoordinationTestsFormViewModel.ts
//$1C50BAB6
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { BalanceCoordinationTestsForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class BalanceCoordinationTestsFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.BalanceCoordinationTestsFormViewModel, Core';
    @Type(() => BalanceCoordinationTestsForm)
    public _BalanceCoordinationTestsForm: BalanceCoordinationTestsForm = new BalanceCoordinationTestsForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/BalanceCoordinationTestsFormViewModel.cs
//$FA487AB1
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class BalanceCoordinationTestsFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class BalanceCoordinationTestsFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var balanceCoordinationTestsFormService = new BalanceCoordinationTestsFormServiceController();
            balanceCoordinationTestsFormService.BalanceCoordinationTestsFormInternal(this, objectContext);
            return (BaseAdditionalInfo)objectContext.GetObject(this._BalanceCoordinationTestsForm.ObjectID, "BalanceCoordinationTestsForm");
        }
    }
}
